test/mix/tasks/phx.gen.schema_test.exs

```diff
@@ -268,6 +268,22 @@ defmodule Mix.Tasks.Phx.Gen.SchemaTest do
         end
       end
     end
+
+    in_tmp_project "uses defaults from generators configuration (binary_id) with Migration Module set", fn ->
```

Comment:
Would it make sense to move this to a different test? This doesn't seem to be related to the `:generators` configuration 🙂 


```elixir
test "generates migrations with a custom migration module", config do
  in_tmp_project config.test, fn ->
    try do
      Application.put_env(:ecto_sql, :migration_module, MyCustomApp.MigrationModule)

      Gen.Schema.run(~w(Blog.Post posts))        

      assert [migration] = Path.wildcard("priv/repo/migrations/*_create_posts.exs")

      assert_file migration, fn file ->
        assert file =~ "use MyCustomApp.MigrationModule"
        assert file =~ "create table(:posts) do"
      end
    after
      Application.delete_env(:ecto_sql, :migration_module)
    end
  end
end
```

I've also moved the cleanup of the Application env to an `after` clause; that way, if `Gem.Schema.run/1` fails, other tests don't get affected.

Reply:
I have updated the tests to reflect your points.  Thank you!